pkg/api/mlflow/dao/convertors/log.go

```diff
@@ -72,3 +78,22 @@ func ConvertLogBatchRequestToDBModel(
 	}
 	return metrics, params, tags, nil
 }
+
+// ConvertLogBatchRequestToContextDBModel converts request.LogBatchRequest into []*models.Context model.
+func ConvertLogBatchRequestToContextDBModel(req *request.LogBatchRequest) ([]*models.Context, error) {
```

Comment:
I'm a little confused why you separately create a slice of models.Context and then provide the slice to other converters as an argument -- couldn't you just FirstOrCreate the context at the place where the Metric is created?